docs/api/ctx.md

```diff
@@ -184,6 +184,35 @@ app.Get("/", func(c *fiber.Ctx) error {
 })
 ```
 
+## AutoFormat
+
+Performs content-negotiation on the [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) HTTP header. It uses [Accepts](ctx.md#accepts) to select a proper format.
+
+:::info
+If the header is **not** specified or there is **no** proper format, **text/plain** is used.
+:::
+
+```go title="Signature"
+func (c *Ctx) AutoFormat(body interface{}) error
+```
+
+```go title="Example"
+app.Get("/", func(c *fiber.Ctx) error {
+  // Accept: text/plain
+  c.AutoFormat("Hello, World!")
+  // => Hello, World!
+
+  // Accept: text/html
+  c.AutoFormat("Hello, World!")
+  // => <p>Hello, World!</p>
+
+  // Accept: application/json
+  c.AutoFormat("Hello, World!")
+  // => "Hello, World!"
```

Comment:
Is this the right output?

Reply:
yes, it's a bare string value, which is valid json; note the double quotes.  Perhaps the example could be more illustrative.

Reply:
Yeah, the officisl express docs use a simple json with key/value pair.